scipy/stats/_qmc.py

```diff
@@ -898,6 +988,151 @@ def random(self, n: IntNumber = 1) -> np.ndarray:
         return samples  # type: ignore[return-value]
 
 
+class OptimalLatinHypercube(QMCEngine):
+    """Optimal Latin hypercube sampling (OLHS).
```

Comment:
```suggestion
    """Greedy optimization of a Latin hypercube sample (OLHS).
```
I think we should mention it's "greedy" somewhere, and I would avoid stating that it's "optimal" because it's not even guaranteed to be locally optimal.